config/boards/rock-5a.wip

```diff
@@ -0,0 +1,28 @@
+# Rockchip RK3588S octa core 4/8/16GB RAM SoC eMMC USB3 USB-C GbE
```

Comment:
USB-C on rock5a is only for power supply, so no need to define `USB-C` here.

Reply:
Dropped `rock-5a` from this PR, we should do it separately.